package-lock.json

```diff
@@ -1,12232 +1,8 @@
 {
   "name": "@briza/illogical",
   "version": "1.4.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Should this be changed? iirc if the local version of npm is out of date this will be changed

@davidhorak for more insight

Reply:
@csimonovsky do you have latest node?

Reply:
I needed to update npm, but im using node v15.3.0

Reply:
c1e5fca